alluxio-ui/master/src/containers/App/App.test.tsx

```diff
@@ -0,0 +1,11 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
```

Comment:
Is this file necessary?

Reply:
That's where the test for App.tsx will be. I like to put test files alongside the thing they test instead of in a separate tests directory. I think it makes things easier in the long run.